src/workloads/scale/ContentionTTLDeletions.yml

```diff
@@ -63,38 +51,183 @@ Actors:
+
+- Name: CreateIndexes
+  Type: RunCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *NumPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *db
+        Operations:
+        - OperationMetricsName: CreateNumFieldIndex
+          OperationName: RunCommand
+          OperationCommand:
+            createIndexes: &otherColl otherColl
+            indexes:
+            - key:
+                numField: 1
+              name: numField
+        - OperationMetricsName: CreateStringFieldIndex
+          OperationName: RunCommand
+          OperationCommand:
+            createIndexes: *otherColl
+            indexes:
+            - key:
+                a: 1
+              name: string
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Database: *db
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *NumPhases
+      PhaseConfig:
+        Threads: 1
+        Repeat: 1
+
+- Name: UpdateOtherColl
+  Type: CrudActor
+  Database: *db
+  Threads: &numThreads 50
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *NumPhases
+      PhaseConfig:
+        Duration: &phaseDuration 10 minutes
+        Collection: *otherColl
+        Operations:
+        - OperationName: updateOne
+          OperationCommand:
+            Filter: &filter
+              numField: {^RandomInt: {min: 0, max: 1000}}
+            Update:
+              $inc: {counter: 1}
+            OperationOptions:
+              WriteConcern: *WMajorityJFalse
+
+- Name: RemoveOtherColl
+  Type: CrudActor
+  Database: *db
+  Threads: *numThreads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *NumPhases
+      PhaseConfig:
+        Duration: *phaseDuration
+        Collection: *otherColl
+        Operations:
+        - OperationName: deleteOne
+          OperationCommand:
+            Filter: *filter
+          OperationOptions:
+            WriteConcern: *WMajorityJFalse
+
+- Name: InsertOtherCollSmall
+  Type: CrudActor
+  Database: *db
+  Threads: &halfThreads 25
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *NumPhases
+      PhaseConfig:
+        Duration: *phaseDuration
+        Collection: *otherColl
+        Operations:
+        - OperationName: insertOne
+          OperationCommand:
+            Document: *SmallDoc
+          OperationOptions:
+            WriteConcern: *WMajorityJFalse
+
+- Name: InsertOtherCollLarge
+  Type: CrudActor
+  Database: *db
+  Threads: *halfThreads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *NumPhases
+      PhaseConfig:
+        Duration: *phaseDuration
+        Collection: *otherColl
+        Operations:
+        - OperationName: insertOne
+          OperationCommand:
+            Document: *LargeDoc
+          OperationOptions:
+            WriteConcern: *WMajorityJFalse
+
+- Name: InsertTTLColl-Small
+  Type: RunCommand
```

Comment:
What are your thoughts on using the CrudActor for these two as well to be consistent with the earlier actors?